basestation/requirements.txt

```
@@ -3,15 +3,14 @@ cffi==1.11.5
 click==6.7        
 Flask==1.1.1     
 Flask-API==2.0
-Flask-Cors==3.0.8
 Flask-SQLAlchemy==2.4.1    
 itsdangerous==0.24
 Jinja2==2.10
```

I got a warning about updating Jinja2 when trying to run this, so we might need to add this to the requirements. It might just be an issue with my machine though.

Console output:
```
flask 1.1.1 requires Jinja2>=2.10.1, but you'll have jinja2 2.10 which is incompatible.
```

```suggestion
Jinja2==2.10.1
```

---

Ooooh okay I'll update that, good catch!